services-custom/s3-transfer-manager/src/main/java/software/amazon/awssdk/transfer/s3/internal/S3CrtAsyncHttpClient.java

```diff
@@ -67,6 +67,7 @@ private S3CrtAsyncHttpClient(Builder builder) {
                                  .withCredentialsProvider(s3NativeClientConfiguration.credentialsProvider())
                                  .withClientBootstrap(s3NativeClientConfiguration.clientBootstrap())
                                  .withPartSize(s3NativeClientConfiguration.partSizeBytes())
+                                 .withComputeContentMd5(true)
```

Comment:
We had to make this change because CRT has pushed fixes to honor this config in latest version. In previous versions, MD5 is always enabled and computeContentMd5 is default to false in the CRT Java binding. We will disable it once we have flexible checksum support.

Reply:
Is it imported to brazil, or will this block our build to live?

Reply:
It should not block our build to live because this config is available in previous CRT version as well (just not honored). I'll check with CRT team to see when they will do an import.